docs/patterns/components/Pagination/index.js

```diff
@@ -130,7 +124,56 @@ const PaginationDemo = () => {
             defaultValue: "'bottom'"
           }
         }}
-      />
+      >
+        <h2>Using the usePagination Hook</h2>
+        <Code>
+          {`const { pagination, pageStatus } = usePagination({
+  totalItems: 111,
+  itemsPerPage: 25
+});
+return <Pagination {...pagination} />;`}
+        </Code>
+        <div className="Demo-props">
+          <h3>usePagination Options</h3>
+          <PropDocs
+            docs={{
+              totalItems: {
+                type: 'number',
+                description:
+                  'The total number of items being rendered (not the total number of pages).',
+                required: true
+              },
+              itemsPerPage: {
+                type: 'number',
+                description: 'The total number of items per page.',
+                required: false,
+                default: 10
+              },
+              initialPage: {
+                type: 'number',
+                description:
+                  'The (1-based) number of the page that pagination will start on.',
+                required: false,
+                default: 1
+              }
+            }}
+            defaultProps={undefined}
+          />
+        </div>
+        <p>
+          <span>
```

Comment:
Nit: This doesn't get converted to markdown, so you may want to wrap the variables with `<code>`. Additionally, are the opening `{` and `}` brackets needed?

Reply:
Yeah, I felt like the `code` tag didn't stick out too well, but I can make that update.